assign_rights/mixins/authmixins.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
+from django.urls import reverse_lazy
+
+
+class LoggedInMixinDefaults(LoginRequiredMixin):
+    """Sets basic login_url for mixin defaults"""
+    login_url = reverse_lazy("login")
+
+
+class UserMixin(LoggedInMixinDefaults, UserPassesTestMixin):
+    """Overwrites the mixin defaults"""
+    authenticated_redirect_url = reverse_lazy("login")
+
+    def test_func(self):
+        """Checks whether a user is a superuser in the application"""
+        if self.request.user.is_superuser:
+            return True
+        return False
```

Comment:
Doesn't this mean that only superusers are going to be allowed to do things in the application?

Reply:
In my local testing, that wasn't happening, however I could have set something else up incorrectly. But the argument for it being there isn't great either, so I just removed that section for now since it wasn't doing anything. Application works as expected without it.